app/frontend/src/setupTests.ts

```diff
@@ -4,16 +4,27 @@
 // learn more: https://github.com/testing-library/jest-dom
 import "@testing-library/jest-dom/extend-expect";
 
+import { server } from "test/restMock";
+
 import user from "./test/fixtures/defaultUser.json";
 
 jest.mock("./service");
 
 global.defaultUser = user;
 global.localStorage = createLocalStorageMock();
 
+beforeAll(() => {
+  server.listen();
```

Comment:
I imagine these will probably only be used in a few specific tests since most of our API is in gRPC still? I think it might be better to scope the set up code of this only in tests that needs the server rather than spinning up for every test suite/file